### ISSUE TITLE
PMO: build.js starts ios emulator when android is selected

### ISSUE THREAD

**Issue by matthewp** [donejs/donejs#481]
Generator puts this in:

```
if(buildCordova) {
  buildPromise.then(stealCordova.build).then(stealCordova.ios.emulate);
}
```

Even if you had selected Android.

**Comment by matthewp** [comment]
Issue: https://github.com/donejs/donejs-cordova/issues/10

**Comment by matthewp** [comment]
This is fixed in donejs-cordova 0.1.3
